engine/src/state_chain_observer/client.rs

```diff
@@ -910,7 +910,7 @@ impl SignedExtrinsicClientBuilderTrait for SignedExtrinsicClientBuilder {
 						)?
 						.submit_and_watch()
 						.await?
-						.wait_for_in_block()
```

Comment:
they removed this in 0.33 as it is unreliable

Reply:
@AlastairHolmes FYI details are in this PR: https://github.com/paritytech/subxt/pull/1237

Could this be related to the SC Client hash mismatch issues we are seeing?

Reply:
We should probably increase the timeout its only 30 seconds